agent/roles/manifests/tasks/install-config.yml

```diff
@@ -7,6 +7,13 @@
   template:
     src: "templates/install-config_yaml.j2"
     dest: "{{ install_path }}/install-config.yaml"
+  when: platform_type != "baremetal"
+
+- name: write the install-config.yaml
+  template:
+    src: "templates/install-config_hosts_yaml.j2"
```

Comment:
I think the name of the template should contains the platform type, ie `install-config_baremetal_yaml.j2`

Reply:
Good point, renamed.